src/plugins/intel_cpu/tests/unit/graph/merge_transpose_reorder_test.cpp

```diff
@@ -305,4 +306,104 @@ INSTANTIATE_TEST_SUITE_P(smoke_Run_MergeTransposeReorderWithReshape,
                          ::testing::Combine(::testing::ValuesIn(input_shapes_with_reshape),
                                             ::testing::ValuesIn(test_params_with_reshape)));
 
-}  // namespace
\ No newline at end of file
+}  // namespace
+
+TEST(MergeTransposeReorderWithReshape_InplaceConflict, smoke_Run_MergeTransposeReorderWithReshape_InplaceConflict) {
+    /*  Initial graph:
+                  Parameter(Layout: nspc)
+                   /                 \
+                Reshape             Dummy1(Layout: nspc, LOOK_UP INPLACE)
+                   |                   |
+               Transpose             Result1
+                   |
+               Dummy0(LOOK_UP INPLACE)
+                   |
+                Result0
+
+        In CreateGraph():
+            1. Reorder will be inserted between Parameter -> Reshape to convert layout
+            2. Reorder -> Reshape -> Transpose will be merged to Reorder
```

Comment:
This reorder is expected to be "optimized", thus non executable, right? If so, can we check in the test as well?

Reply:
done